modules/cluster-logging-rn-5.5.5.adoc

```diff
@@ -0,0 +1,87 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-5_{context}"]
+= Logging 5.5.5
+This release includes link:https://access.redhat.com/errata/RHSA-2022:8781[OpenShift Logging Bug Fix Release 5.5.5].
+
+[id="openshift-logging-5-5-5-bug-fixes"]
+== Bug fixes
+* Before this update, Kibana had a fixed `24h` OAuth cookie expiration time, which resulted in 401 errors in Kibana whenever the `accessTokenInactivityTimeout` field was set to a value lower than `24h`. With this update, Kibana's OAuth cookie expiration time synchronizes to the `accessTokenInactivityTimeout`, with a default value of `24h`. (link:https://issues.redhat.com/browse/LOG-3305[LOG-3305])
+
+* Before this update, vector parsed the message field when JSON  parsing was enabled without also defining `structuredTypeKey` or `structuredTypeName` values. With this update, a value is required for either  `structuredTypeKey` or `structuredTypeName` when writing structured logs to Elasticsearch. (link:https://issues.redhat.com/browse/LOG-3284[LOG-3284])
```

Comment:
Should vector be have initial caps Vector? 

https://docs.openshift.com/container-platform/4.11/logging/cluster-logging-loki.html#cluster-logging-vector-enable_cluster-logging-loki